hazelcast/proxy/vector_collection.py

```diff
@@ -29,29 +29,117 @@
 
 
 class VectorCollection(Proxy["BlockingVectorCollection"]):
+    """VectorCollection contains documents with vectors.
+
+    Concurrent, distributed, observable and searchable vector collection.
+    The vector collection can work both async(non-blocking) or sync(blocking).
```

Comment:
```suggestion
    The vector collection can work both async (non-blocking) or sync (blocking).
```

Reply:
That's directly from the `IMap` docs, so I'll keep that for consistency.

Reply:
IMO we do not need typo-level consistency, but we can keep it as is